lib/elixir/lib/calendar/iso.ex

```diff
@@ -41,12 +41,35 @@ defmodule Calendar.ISO do
   @type year :: -9999..9999
   @type month :: 1..12
   @type day :: 1..31
+  @type hour :: 0..23
+  @type minute :: 0..59
+  @type second :: 0..59
+
+  @type day_of_era :: Calendar.day_of_era()
```

Comment:
this type is added in this PR: https://github.com/elixir-lang/elixir/pull/9511
please do not merge until the other PR is merged, or modify this type